src/main/java/com/slisenko/education/html/PageDownloader.java

```diff
@@ -0,0 +1,48 @@
+package com.slisenko.education.html;
+
+import java.io.*;
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+
+/**
+ * Class for downloading resources from the internet
+ */
+public class PageDownloader {
+
+    private final int SIZE_BUF = 256;
+
+    /**
+     * Downloads a resource from the internet and saves it in a file
+     * @param urlStr Link to the resource
+     * @param pathStr File path
+     * @return Returns true if successful, otherwise returns false
+     */
+    public boolean download(String urlStr, String pathStr) {
```

Comment:
no need to name it "urlStr", just url and path

Reply:
urlStr and pathStr changed to link and path